android/clientmanager/src/main/java/io/mosip/registration/clientmanager/service/Biometrics095Service.java

```diff
@@ -109,6 +134,15 @@ public List<BiometricsDto> handleRCaptureResponse(Modality modality, InputStream
                         false,
                         1, 0,
                         captureDto.getQualityScore()));
+
+                if(RegistrationConstants.ENABLE.equalsIgnoreCase(this.globalParamRepository
+                        .getCachedStringGlobalParam(RegistrationConstants.DEDUPLICATION_ENABLE_FLAG))) {
```

Comment:
we should set this flag to shared preferences, and update the value whenever a sync is done and use the value from there rather than querying db each time

Reply:
Added flag to shared preferences instead for fetching from db everytime